src/service/WordService.js

```diff
@@ -27,8 +27,12 @@ export default class WordService {
   }
 
   async getTodayWord() {
+    const currUserId = JSON.parse(localStorage.getItem(LOCALSTORAGE_KEY));
```

Comment:
the `userId` should be passed to `WordService` from `InsertWordUI`, which in turn is passed from `insertWord.js` (the bridge among different resources).
Strictly speaking, how we obtain the `userId` should not be part of the `WordService` responsibility